crates/lsp/src/lsp.rs

```diff
@@ -358,12 +358,19 @@ impl LanguageServer {
             workspace_folders,
             cx,
             move |notification| {
-                log::info!(
-                    "Language server with id {} sent unhandled notification {}:\n{}",
-                    server_id,
-                    notification.method,
-                    serde_json::to_string_pretty(&notification.params).unwrap(),
-                );
+                if notification.id.is_some() {
```

Comment:
Ironically, not only this PR bloats the logs with json, but hides the information about methods that were never handled, that is very odd.